_includes/head.html

```diff
@@ -0,0 +1,15 @@
+<head>
+  <meta charset="utf-8">
+  
+  <title>{% if page.title %} {{ page.title }} | {% endif %} {{ site.name }}</title>
```

Comment:
I don't know if it's intended, but there would be double spacing between a page title and the site name. Super minor, but... slightly inconsistent.